src/file-attachments.js

```diff
@@ -31,11 +31,14 @@ export default function findFileAttachments(cell) {
           );
         }
 
-        references.add(
-          args[0].type === "Literal"
-            ? args[0].value
-            : args[0].quasis[0].value.cooked
-        );
+        const fileReference =
+          args[0].type === "Literal" ? args[0].value : args[0].quasis[0].value.cooked;
+
+        if (references.has(fileReference)) {
+          references.get(fileReference).push(args[0].start);
```

Comment:
Relying on the length `fileReference` is potentially dangerous - a file reference like

```
File("\u0045llo")
```

Will get an `args[0].value` of "Ello", but the length of the string in code would be 11 chars instead of 4. Either we'll need to keep the `raw` value around, or store both start and end indexes.

Reply:
got it -- thanks! 👍